alacritty_terminal/src/tty/windows/conpty.rs

```diff
@@ -20,7 +20,7 @@ use crate::tty::windows::child::ChildExitWatcher;
 use crate::tty::windows::{cmdline, win32_string, Pty};
 
 // Adjust to taste. Too high means higher memory consumption, too low means over-synchronization.
```

Comment:
This comment needs updating.

Reply:
I also meant that we can use `8 * BUFFER` or something based on that if it's "better".

Reply:
We could check with `vtebench` whether changing anything results in change in performance.

Reply:
Could you drop this comment?